react-sizeme.d.ts

```diff
@@ -3,29 +3,42 @@ import { Component, ComponentType, ReactNode, ReactElement } from 'react'
 type Omit<T, K> = Pick<T, Exclude<keyof T, K>>
 
 export interface SizeMeProps {
-  size: {
-    width: number | null
-    height: number | null
+  readonly size: {
+    readonly width: number | null
+    readonly height: number | null
   }
 }
 
 export interface SizeMeOptions {
-  monitorWidth?: boolean
-  monitorHeight?: boolean
-  monitorPosition?: boolean
-  refreshRate?: number
-  refreshMode?: 'throttle' | 'debounce'
-  noPlaceholder?: boolean
+  readonly monitorWidth?: boolean
+  readonly monitorHeight?: boolean
+  readonly monitorPosition?: boolean
+  readonly refreshRate?: number
+  readonly refreshMode?: 'throttle' | 'debounce'
+  readonly noPlaceholder?: boolean
 }
 
 export interface SizeMeRenderProps extends SizeMeOptions {
-  children: (props: SizeMeProps) => ReactElement
+  readonly children: (props: SizeMeProps) => ReactElement
```

Comment:
Same here, this is user provided, and won't get exposed in a manner that the user could modify it, so the `readonly` isn't really useful in this case.